templates/repo/issue/view_content/add_reaction.tmpl

```diff
@@ -4,16 +4,16 @@
 		{{svg "octicon-plus-small" 16}}
 		{{svg "octicon-smiley" 16}}
 	</a>
-	<div class="menu has-emoji">
+	<div class="menu">
 		<div class="header">{{ .ctx.i18n.Tr "repo.pick_reaction"}}</div>
 		<div class="divider"></div>
 		{{range $value := AllowedReactions}}
 			{{if eq $value "hooray"}}
-				<div class="item" data-content="hooray">:tada:</div>
+				<div class="item reaction" data-content="hooray">{{ReactionToEmoji "tada"}}</div>
```

Comment:
If we support alias now the whole if statement is pointles and can be removed

Reply:
can we migrate them ?

-> I think this is worth it's own pull

Reply:
fixed -- add custom aliases before we generate the files